tyk-hybrid/templates/deployment-gw-repset.yaml

```diff
@@ -92,11 +89,25 @@ spec:
           - name: TYK_GW_SLAVEOPTIONS_BINDTOSLUGSINSTEADOFLISTENPATHS
             value: "{{ default "false" .Values.gateway.rpc.bindToSlugs }}"
           - name: TYK_GW_SLAVEOPTIONS_SSLINSECURESKIPVERIFY
-            value: "{{ .Values.gateway.rpc.sslInsecureSkipVerify }}"  
+            value: "{{ .Values.gateway.rpc.sslInsecureSkipVerify }}"
+        # Enable backward compatibility to chart 0.5.1.
+        {{- if or .Values.gateway.sharding .Values.enableSharding}}
```

Comment:
There seems to be no `.Values.enableSharding` values in values.yml

Reply:
Will resolve this in favour of the comment above.